docs/Events/bpmn-events/import-parse-start.md

```diff
@@ -10,3 +10,25 @@ grand_parent: "Events"
 ---
 
 # import.parse.start
+
+This event is fired by [importXML API](/general/api-reference/bpmn-js/importXML) with data containing `xml` before the import begins. This allows users to hook into and manipulate imported XML.
+
+## Example Usage
+
+```javascript
+bpmnJS.on('import.parse.start', function(data) {
+
+  const { xml } = data; // this is the XML that is about to get imported
+
+  const newXML = replace('StartEvent_1', 'StartEvent_2');
+
+  return newXML; // now this new XML will be imported instead
+});
```

Comment:
If we run this example, the import will be broken due to missing BPMN element for StartEvent_1 DI.

Reply:
This is just an example to demonstrate what this event does. It's hard to tell what is broken since we don't know whats inside the XML anyway. We can instead add a regex replace all kinda thing there but it's not adding more value to it IMO. What would be the alternative that you propose instead of this code?

Reply:
It could be sth like `replace('Order pizza', 'Order sushi')`, but I'm also OK with how it looks now.